static/faq.html

```diff
@@ -291,6 +294,55 @@ <h2 id="default-connections">
                 </li>
             </ul>
 
+            <h2 id="securing-data-at-rest">
+                <a href="#securing-data-at-rest">How does GrapheneOS secure data at rest?</a>
+            </h2>
+            <p>GrapheneOS makes full use of the hardware-bound encryption in modern Android devices.
+            The Titan M, a specially designed hardware security module on Pixel 3 handsets handles
+            validation of the user's pincode or password. Upon getting the correct password, the HSM
+            will calculate an access token from the password, from a secret contained in the
+            confines of the HSM, and from a hash of a two-kilobyte partition at the start of the
+            phone's SSD by means of a key derivation function. The HSM will only be able to release
+            the token to the phone so it can in turn derive the keys to decrypt the files if it gets
+            the correct password.</p>
+
+            <p>This system ensures that an adversary in possession of your phone cannot simply
+            extract the SSD and connect it to a much more powerful system to run guessing attacks on
+            it offline, and instead must go through the specific HSM on the phone. The HSM is
+            about the size of a grain of rice and is difficult to attack. Each file is encrypted
+            with AES-256-XTS, and the filesystem metadata is encrypted with AES-256-HEH.</p>
```

Comment:
The vocab is a bit inconsistent; it jumps between "HSM" in this paragraph to "Titan M" in the next paragraph

Reply:
Sure. I'll go and get that...

Reply:
Should be fixed now!